__tests__/app.test.js

```diff
@@ -166,5 +212,117 @@ describe('/api/articles', ()=>{
             })
         })
     })
-    
-})
\ No newline at end of file
+    describe('POST /api/articles/:article_id/comments valid endpoints', ()=>{
+        test('201 Adds a new comment object to the database', ()=>{
+            const newComment = {
+                "username": "butter_bridge",
+                "body": "I can't wait for bed-time"
+            }
+            return request(app)
+            .post("/api/articles/5/comments")
+            .send(newComment)
+            .expect(201)
+            .then(() => {
+                return request(app)
+                .get("/api/articles/5/comments")
+                .then(({body}) => {
+                expect(body.length).toBe(3);
+                });
+            })
+        })
+        test('201 Responds with the posted comment object', ()=>{
+            
+            const postedComment = {
+                "username": "butter_bridge",
+                "body": "I can't wait for bed-time"
+            }
+            return request(app)
+            .post("/api/articles/5/comments")
+            .send(postedComment)
+            .expect(201)
+            .then(({body})=>{
+                expect(body.postedComment).toMatchObject({
+                    "author": "butter_bridge",
```

Comment:
I now see you've done the above comment here, we don't need the above assertion then. Also, we will get more properties back on a posted comment so lets assert against all of those properties and values.